spec/dummy/app/views/styleguide/components.slim

```diff
@@ -0,0 +1,39 @@
+= render 'navbar'
```

Comment:
Shouldn't the navbar be rendered in the layout in order to not having to do it in two places?